fake-cucumber/javascript/.mocharc.local.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
So this is the same as `.mocharc.local.json` but without the require of tsconfig-paths - does it break without this? Does it also mean you have to build dependent modules before running tests now?

Reply:
(Not the end of the world if so, as these will presumably be coming out of the monorepo soon enough, but curious.)

Reply:
Yes, tsconfig-paths does break the build.

Actually if I recall correctly,we already need to build JavaScript modules before anything else